src/main/scala/scala/tools/partest/nest/Runner.scala

```diff
@@ -765,7 +765,7 @@ class SuiteRunner(
 
   def banner = {
     val baseDir = fileManager.compilerUnderTest.parent.toString
-    def relativize(path: String) = path.replace(baseDir, "$baseDir").replace(PathSettings.srcDir.toString, "$sourceDir")
+    def relativize(path: String) = path.replace(baseDir, s"$$baseDir").replace(PathSettings.srcDir.toString, "$sourceDir")
```

Comment:
yeah, I understood and thought about changing it, but decided I like it a little better this way, just because `s` is so much more familiar than `f`. (no strong feeling either way though)